docs/src/index.md

```diff
@@ -0,0 +1,87 @@
+
+- Make building deep learning models more convenient to users already familiar with the MLJ workflow
+
+- Make it easier to apply machine learning techniques provided by MLJ, including: out-of-sample performance evaluation, hyper-parameter optimization, iteration control, and more, to deep learning models
+
+!!! note "MLJFlux Coverage"
+    MLJFlux support is focused on fundamental and widely used deep learning models; sophisticated architectures or techniques such as online learning, reinforcement learning, and adversarial networks are currently beyond its scope. 
+
+Also note that MLJFlux is limited to training models only when all training data fits into memory, though it still supports automatic batching of data.
+
+## Installation
+
+```julia
+import Pkg
+Pkg.activate("my_environment", shared=true)
+Pkg.add(["MLJ", "MLJFlux", "Flux"])
+```
+You only need `Flux` if you need to build a custom architecture or experiment with different optimizers, loss functions and activations.
+
+## Quick Start
+First load and instantiate mode:
+```@example
+using MLJ, Flux, MLJFlux
+import RDatasets
+
+# 1. Load Data
+iris = RDatasets.dataset("datasets", "iris");
+y, X = unpack(iris, ==(:Species), colname -> true, rng=123);
+
+# 2. Load and instantiate model
+NeuralNetworkClassifier = @load NeuralNetworkClassifier pkg="MLJFlux"
+clf = NeuralNetworkClassifier(
+    builder=MLJFlux.MLP(; hidden=(5,4), σ=Flux.relu),
+    optimiser=Flux.ADAM(0.01),
+    batch_size=8,
+    epochs=100, 
+    acceleration=CUDALibs()         # For GPU support
+    )
+
+# 3. Wrap it in a machine in fit
+mach = machine(clf, X, y)
+fit!(mach)
+
+# 4. Evaluate the model
+cv=CV(nfolds=5)
+evaluate!(mach, resampling=cv, measure=accuracy) 
+```
+As you can see we were able to use MLJ functionality (i.e., cross validation) with a Flux deep learning model. All arguments provided also have defaults.
+
+Notice that we were also able to define the neural network in a high-level fashion by only specifying the number of neurons per each hidden layer and the activation function. Meanwhile, `MLJFlux` was able to infer the input and output layer as well as use a suitable default for the loss function and output activation given the classification task. Notice as well that we did not need to implement a training or prediction loop as in `Flux`.
+
+## Basic idea
+
+As in the example above, any MLJFlux model has a `builder` hyperparameter, an object encoding
+instructions for creating a neural network given the data that the
+model eventually sees (e.g., the number of classes in a classification
+problem). While each MLJ model has a simple default builder, users
+may need to define custom builders to get optimal results,
+and this will require familiarity with the [Flux
+API](https://fluxml.ai/Flux.jl/stable/) for defining a neural network
+chain.
+
+In the future MLJFlux may provide a larger assortment of canned
+builders. Pull requests introducing new ones are most welcome.
```

Comment:
```suggestion
```

Reply:
Somehow I remember removing this before making PR ready for review 🤷.